.github/workflows/ci-disable-gtest.yml

```diff
@@ -5,7 +5,13 @@
 #   * Disables gtest.
 
 name: CI Disable GTest
-on: [push, pull_request]
+on:
+  push
```

Comment:
I didn't see an example or reference for merging two sections this way (no trailing ':') here.

Reply:
I saw the no trailing `:` here: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push . I added `:` just to be more YAML like .